### ISSUE TITLE
Electron 7.1.7: accessibilityLabel does not appear for touch bar button

### ISSUE THREAD

**Issue by bpasero** [electron/electron#21715]
### Issue Details

* **Electron Version:**
  * 7.1.7
* **Operating System:**
  * macOS

### Expected Behavior
With https://github.com/electron/electron/pull/20454 there should be a new `accessibilityLabel` property on touchbar items. 

### Actual Behavior
There is no such property in `electron.d.ts`

### To Reproduce
Simply depend on `"electron": "7.1.7"` in `package.json` and check the types. 

//cc @MarshallOfSound @deepak1556

**Comment by jacobq** [comment]
I don't think that PR has landed in 7.x (or will be backported), but I [see it in v8.0.0-beta.1](https://electronjs.org/releases/beta#release-notes-for-v800-beta1).

**Comment by MarshallOfSound** [comment]
Yeah that property was not sent back to 7.x, it's semver/minor and therefore only expected in 8.x
